### PR TITLE
changed SHOCK upload in unit tests to DataFileUtil.file_to_shock()

### PR DESCRIPTION
**Pull request by Yzm1234**
changed SHOCK upload in unit tests to DataFileUtil.file_to_shock()

### CODE REVIEW

**Review by dcchivian — APPROVED** [review]
unit tests are now all passing.  version in kbase.yml is bumped and RELEASE_NOTES.md are updated.  I approve.

File: .travis.yml [hunk 1]
```diff
@@ -20,10 +20,22 @@ before_install:
 install:
 - pip install coveralls
 - git clone https://github.com/kbase/jars
-- git clone https://github.com/kbase/kb_sdk
-- cd kb_sdk
-- make
-- export PATH=$(pwd)/bin:$PATH
-- cd ../
+#- git clone https://github.com/kbase/kb_sdk
```
Comment (bio-boris) [inline .travis.yml:23]: You can just delete this commented code

**Review by bio-boris — APPROVED** [review]
LGTM

**Comment by jsfillman** [comment]
:shipit:

**Review by jsfillman — APPROVED** [review]

**Comment by dcchivian** [comment]
codeowners and other reviewers are passing.  all tests are passing.  I'm merging and registering on CI.

**Pull request merged**